src/library/managers/ImageManager.js

```diff
@@ -0,0 +1,96 @@
+/* ImageManager.js
+KC3改 Images Manager
+
+Manages images, since they are too thicc for webstore
+Saves and loads to db
+
+Does: intercept jquery source changes
+Does not: edit already-existing src tags in HTML 
+*/
+(function(){
+	"use strict";
+	
+	window.KC3ImageManager = {
+		errorImage : "../../../../assets/img/ui/empty.png",
+		cdn : "",
+		//imageCache : {},
+
+		init :function() {
+			var self = this;
+			var originalAttr = jQuery.fn.attr;
+
+			self.cdn = "https://raw.githubusercontent.com/KC3Kai/KC3Kai/" + (localStorage.lastestCommit || "1809dc3f110f694c52f9d89dc9404c726ef1f02e") + "/src/";
+			self.checkForUpdate();
+
+			jQuery.fn.attr = function() {
+				var arg = arguments;
+				var jQueryObject = this;
+
+				if(arguments.length > 1 && arguments[0] === "src") {
+					var url = arg[1].replace(/(\.\.\/)+/g, "");
```

Comment:
might double check: `src` starts with `chrome-extension://` and (or) `/assets/img`